client/components/Configuration/ImportExport.jsx

```diff
@@ -49,22 +40,46 @@ export default connectContainer(class extends Component {
     }
   }
 
+  confirmImport = () => {
+    const preview = this.props.preview;
+    this.props.importConfig(preview.toJSON())
+  }
+
+  closeImport = () => {
+    this.refs.file.value = '';
```

Comment:
Refs are no longer encouraged: https://facebook.github.io/react/docs/refs-and-the-dom.html
do we really need to add them here? :S